Add files via upload

jacovandermolen opened the pull request.

RolandMelkert commented:
The 60481 file looks ok but it isn't needed. 

The missing snapping is caused by a recent 'moved to'. This caused the previous used subfile to be orphaned. So the fix is to move/rename 60481s01.dat to 60481as01.dat file to the new one..

I fixed it in main.

RolandMelkert commented:
I've deleted the 3 modified ones as they conflicted with earlier changes. 
I also corrected the headers, please use LDCad 1.7's auto shadow header feature.

jacovandermolen commented:
> I've deleted the 3 modified ones as they conflicted with earlier changes. I also corrected the headers, please use LDCad 1.7's auto shadow header feature.

OK. Sorry, my mistake to not use 1.7. Will do!
